src/test/java/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -129,56 +129,64 @@ public void testTimeParses() {
 
         TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
 
-        testTime("10:00", 1000 * 60 * 60 * 10 - getOffset());
-        testTime("10:00Z", 1000 * 60 * 60 * 10);
+        Calendar midnight = Calendar.getInstance();
```

Comment:
This would commonly be called `startOfDay` rather than `midnight` (see [Java8's `LocalDate` API](https://docs.oracle.com/javase/8/docs/api/index.html))